pkg/monitor/dynkeepalived.go

```diff
@@ -98,6 +98,19 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 	var appliedConfig, curConfig, prevConfig *config.Node
 	var configChangeCtr uint8 = 0
 
+	clusterName, _, err := config.GetClusterNameAndDomain(kubeconfigPath, clusterConfigPath)
+	if err != nil {
+		return err
+	}
+
+	if exists, err := needLease(cfgPath); err != nil {
+		return err
```

Comment:
I dont think u should fail - it will break backwards competability. I think no matter what in this stage, u should proceed without dhcp lease and write a log.
@celebdor what do u think